lib/JobHandler.php

```diff
@@ -50,7 +51,7 @@ class JobHandler
 	public $endTime;
 
 	/**
-	 * @var Job Instance of the class performing work for this job.
+	 * @var Job|null Instance of the class performing work for this job.
 	 */
 	private $instance;
```

Comment:
I don't think this should support `null`. From the code it looks like that was mostly meant as an `isset()` check, so we can treat it as such.

Reply:
Should I adjust the check to `isset` then?